src/assets/stylesheets/index.scss

```diff
@@ -1187,7 +1188,8 @@ section:focus >,
 // -- general form-control -- //
 
 .form-label {
-  font-family: var(--jr-headline-font-family), sans-serif;
+  font-family: var(--bs-body-font-family), sans-serif;
+  font-weight: var(--bs-body-font-weight);
```

Comment:
How about `font-weight: bolder` here?

Reply:
We can delete family and weight here. 'Bolder' is a deign decision. From my point of view no.